src/ReactiveUI.XamForms/ReactiveMasterDetailPage.cs

```diff
@@ -13,6 +14,7 @@ namespace ReactiveUI.XamForms
     /// <typeparam name="TViewModel">The type of the view model.</typeparam>
     /// <seealso cref="Xamarin.Forms.MasterDetailPage" />
     /// <seealso cref="ReactiveUI.IViewFor{TViewModel}" />
+    [Obsolete("ReactiveMasterDetailPage is obsolete as of version 5.0.0. Please use FlyoutPage instead.")]
```

Comment:
This is not accurate.  Xamarin isn't forcing you to use Shell yet.

https://docs.microsoft.com/en-us/xamarin/xamarin-forms/release-notes/5.0/5.0.0#masterdetailpage-renamed

Reply:
@RLittlesII This is the warning shown in the base MasterDetailPage of ReactiveMasterDetailPage, I just brought the visibility up to replicate the base. With the introduction of ReactiveFlyoutPage I felt this was reasonable to expose.

Reply:
Would you like this removed to resolve this, or are you happy with my reasoning? 
Perhaps it should say ReactiveFlyoutPage instead of FlyoutPage

Reply:
Your reasoning is fine.  I just dislike that we have to make changes based on the dependencies we consume.  My thought was if it's just a name change and not a functionality change, why does it matter?  We should be able to swap the dependency and not see a disruption in the features.  Then our consumers don't have to change their applications.